### DISCUSSION TITLE
Need for improved algorithm for fitting horizontal deflection parameters

### DISCUSSION THREAD

**Discussion by misi9170** [NREL/flasc#142] [General]
FLASC v1.4 introduced a FLORIS model tuning package with a preliminary implementation for tuning various wake model parameters, particularly focused the Empirical Gaussian FLORIS model. We have been finding that these methods work reasonably well for fitting the velocity deficit model, but the fitting approach is not always able to fit the horizontal deflection parameters (which manifest when a turbine has a yaw offset).

This is best demonstrated in examples_smarteole/08_emgauss_tuning_day_night.ipynb, introduced in #127. For the day time data, the best fit horizontal deflection parameter is near zero. This is because the yawing of the upstream turbine reduces its thrust, which effectively produces enough of an uplift at the waked turbine to match the data without requiring any deflection.

**Comment by paulf81** [comment]
Agree!

**Comment by Bartdoekemeijer** [comment]
I see how these can be tough. Indeed, when just looking at power downstream, how do we distinguish lateral deflection from vertical deflection from added turbulence-induced recovery from another aerodynamic effect? I wouldn't be surprised if finally we find that we need flow field measurements or wake center deflection measurements to calibrate these parameters accurately. In practice, this would require LES simulations or lidar measurements in the field. Perhaps power measurements by themselves do not give enough detail to really estimate these effects? Just speculation but worth considering we may just not have enough information in the measurements to estimate every parameter accurately.

**Reply by paulf81** [reply]
Agree with all of this, I would add the extra point that playing around with this example: https://github.com/NREL/flasc/blob/main/examples_artificial_data/03_energy_ratio/07_wake_steering_total_uplift.py convinced me that uplift measurements are pretty susceptible to noise in the wind direction measurement
